{"filter":false,"title":"rspec.log","tooltip":"/bookers3/rspec.log","undoManager":{"mark":0,"position":0,"stack":[[{"start":{"row":262,"column":0},"end":{"row":593,"column":0},"action":"insert","lines":["\u001b[32m    投稿一覧画面\u001b[0m","\u001b[32m    投稿詳細画面\u001b[0m","  アイコンのテスト","    トップ画面","\u001b[32m      本のアイコンが表示される\u001b[0m","    アバウト画面","\u001b[32m      本のアイコンが表示される\u001b[0m","    ヘッダー: ログインしていない場合","\u001b[32m      Homeリンクのアイコンが表示される\u001b[0m","\u001b[32m      Aboutリンクのアイコンが表示される\u001b[0m","\u001b[32m      Sign upリンクのアイコンが表示される\u001b[0m","\u001b[32m      Log inリンクのアイコンが表示される\u001b[0m","    ヘッダー: ログインしている場合","\u001b[32m      Homeリンクのアイコンが表示される\u001b[0m","\u001b[31m      Usersリンクのアイコンが表示される (FAILED - 25)\u001b[0m","\u001b[31m      Booksリンクのアイコンが表示される (FAILED - 26)\u001b[0m","\u001b[32m      Log outリンクのアイコンが表示される\u001b[0m","    サイドバー","\u001b[32m      ユーザ一覧画面でレンチアイコンが表示される\u001b[0m","\u001b[32m      ユーザ詳細画面でレンチアイコンが表示される\u001b[0m","\u001b[32m      投稿一覧画面でレンチアイコンが表示される\u001b[0m","\u001b[32m      投稿詳細画面でレンチアイコンが表示される\u001b[0m","","Failures:","","  1) [STEP2] ユーザログイン後のテスト 投稿一覧画面のテスト サイドバーの確認 titleフォームが表示される","     \u001b[31mFailure/Error: expect(page).to have_field 'book[title]'\u001b[0m","     \u001b[31m  expected to find field \"book[title]\" that is not disabled but there were no matches\u001b[0m","     \u001b[36m# ./spec/system/02_after_login_spec.rb:76:in `block (4 levels) in <main>'\u001b[0m","","  2) [STEP2] ユーザログイン後のテスト 投稿一覧画面のテスト サイドバーの確認 titleフォームに値が入っていない","     \u001b[31mFailure/Error: expect(find_field('book[title]').text).to be_blank\u001b[0m","     \u001b[31m\u001b[0m","     \u001b[31mCapybara::ElementNotFound:\u001b[0m","     \u001b[31m  Unable to find field \"book[title]\" that is not disabled\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:129:in `find_field'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","     \u001b[36m# ./spec/system/02_after_login_spec.rb:79:in `block (4 levels) in <main>'\u001b[0m","","  3) [STEP2] ユーザログイン後のテスト 投稿一覧画面のテスト サイドバーの確認 bodyフォームが表示される","     \u001b[31mFailure/Error: expect(page).to have_field 'book[body]'\u001b[0m","     \u001b[31m  expected to find field \"book[body]\" that is not disabled but there were no matches\u001b[0m","     \u001b[36m# ./spec/system/02_after_login_spec.rb:82:in `block (4 levels) in <main>'\u001b[0m","","  4) [STEP2] ユーザログイン後のテスト 投稿一覧画面のテスト サイドバーの確認 bodyフォームに値が入っていない","     \u001b[31mFailure/Error: expect(find_field('book[body]').text).to be_blank\u001b[0m","     \u001b[31m\u001b[0m","     \u001b[31mCapybara::ElementNotFound:\u001b[0m","     \u001b[31m  Unable to find field \"book[body]\" that is not disabled\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:129:in `find_field'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","     \u001b[36m# ./spec/system/02_after_login_spec.rb:85:in `block (4 levels) in <main>'\u001b[0m","","  5) [STEP2] ユーザログイン後のテスト 投稿一覧画面のテスト 投稿成功のテスト 自分の新しい投稿が正しく保存される","     \u001b[31mFailure/Error: fill_in 'book[title]', with: Faker::Lorem.characters(number: 5)\u001b[0m","     \u001b[31m\u001b[0m","     \u001b[31mCapybara::ElementNotFound:\u001b[0m","     \u001b[31m  Unable to find field \"book[title]\" that is not disabled\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/actions.rb:91:in `fill_in'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","     \u001b[36m# ./spec/system/02_after_login_spec.rb:94:in `block (4 levels) in <main>'\u001b[0m","","  6) [STEP2] ユーザログイン後のテスト 投稿一覧画面のテスト 投稿成功のテスト リダイレクト先が、保存できた投稿の詳細画面になっている","     \u001b[31mFailure/Error: fill_in 'book[title]', with: Faker::Lorem.characters(number: 5)\u001b[0m","     \u001b[31m\u001b[0m","     \u001b[31mCapybara::ElementNotFound:\u001b[0m","     \u001b[31m  Unable to find field \"book[title]\" that is not disabled\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/actions.rb:91:in `fill_in'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","     \u001b[36m# ./spec/system/02_after_login_spec.rb:94:in `block (4 levels) in <main>'\u001b[0m","","  7) [STEP2] ユーザログイン後のテスト 自分の投稿詳細画面のテスト サイドバーの確認 titleフォームが表示される","     \u001b[31mFailure/Error: expect(page).to have_field 'book[title]'\u001b[0m","     \u001b[31m  expected to find field \"book[title]\" that is not disabled but there were no matches\u001b[0m","     \u001b[36m# ./spec/system/02_after_login_spec.rb:149:in `block (4 levels) in <main>'\u001b[0m","","  8) [STEP2] ユーザログイン後のテスト 自分の投稿詳細画面のテスト サイドバーの確認 titleフォームに値が入っていない","     \u001b[31mFailure/Error: expect(find_field('book[title]').text).to be_blank\u001b[0m","     \u001b[31m\u001b[0m","     \u001b[31mCapybara::ElementNotFound:\u001b[0m","     \u001b[31m  Unable to find field \"book[title]\" that is not disabled\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:129:in `find_field'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","     \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","     \u001b[36m# ./spec/system/02_after_login_spec.rb:152:in `block (4 levels) in <main>'\u001b[0m","","  9) [STEP2] ユーザログイン後のテスト 自分の投稿詳細画面のテスト サイドバーの確認 bodyフォームが表示される","     \u001b[31mFailure/Error: expect(page).to have_field 'book[body]'\u001b[0m","     \u001b[31m  expected to find field \"book[body]\" that is not disabled but there were no matches\u001b[0m","     \u001b[36m# ./spec/system/02_after_login_spec.rb:155:in `block (4 levels) in <main>'\u001b[0m","","  10) [STEP2] ユーザログイン後のテスト 自分の投稿詳細画面のテスト サイドバーの確認 bodyフォームに値が入っていない","      \u001b[31mFailure/Error: expect(find_field('book[body]').text).to be_blank\u001b[0m","      \u001b[31m\u001b[0m","      \u001b[31mCapybara::ElementNotFound:\u001b[0m","      \u001b[31m  Unable to find field \"book[body]\" that is not disabled\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:129:in `find_field'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","      \u001b[36m# ./spec/system/02_after_login_spec.rb:158:in `block (4 levels) in <main>'\u001b[0m","","  11) [STEP2] ユーザログイン後のテスト 自分の投稿詳細画面のテスト 投稿成功のテスト 自分の新しい投稿が正しく保存される","      \u001b[31mFailure/Error: fill_in 'book[title]', with: Faker::Lorem.characters(number: 5)\u001b[0m","      \u001b[31m\u001b[0m","      \u001b[31mCapybara::ElementNotFound:\u001b[0m","      \u001b[31m  Unable to find field \"book[title]\" that is not disabled\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/actions.rb:91:in `fill_in'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","      \u001b[36m# ./spec/system/02_after_login_spec.rb:167:in `block (4 levels) in <main>'\u001b[0m","","  12) [STEP2] ユーザログイン後のテスト ユーザ一覧画面のテスト サイドバーの確認 titleフォームが表示される","      \u001b[31mFailure/Error: expect(page).to have_field 'book[title]'\u001b[0m","      \u001b[31m  expected to find field \"book[title]\" that is not disabled but there were no matches\u001b[0m","      \u001b[36m# ./spec/system/02_after_login_spec.rb:292:in `block (4 levels) in <main>'\u001b[0m","","  13) [STEP2] ユーザログイン後のテスト ユーザ一覧画面のテスト サイドバーの確認 titleフォームに値が入っていない","      \u001b[31mFailure/Error: expect(find_field('book[title]').text).to be_blank\u001b[0m","      \u001b[31m\u001b[0m","      \u001b[31mCapybara::ElementNotFound:\u001b[0m","      \u001b[31m  Unable to find field \"book[title]\" that is not disabled\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:129:in `find_field'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","      \u001b[36m# ./spec/system/02_after_login_spec.rb:295:in `block (4 levels) in <main>'\u001b[0m","","  14) [STEP2] ユーザログイン後のテスト ユーザ一覧画面のテスト サイドバーの確認 bodyフォームが表示される","      \u001b[31mFailure/Error: expect(page).to have_field 'book[body]'\u001b[0m","      \u001b[31m  expected to find field \"book[body]\" that is not disabled but there were no matches\u001b[0m","      \u001b[36m# ./spec/system/02_after_login_spec.rb:298:in `block (4 levels) in <main>'\u001b[0m","","  15) [STEP2] ユーザログイン後のテスト ユーザ一覧画面のテスト サイドバーの確認 bodyフォームに値が入っていない","      \u001b[31mFailure/Error: expect(find_field('book[body]').text).to be_blank\u001b[0m","      \u001b[31m\u001b[0m","      \u001b[31mCapybara::ElementNotFound:\u001b[0m","      \u001b[31m  Unable to find field \"book[body]\" that is not disabled\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:129:in `find_field'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","      \u001b[36m# ./spec/system/02_after_login_spec.rb:301:in `block (4 levels) in <main>'\u001b[0m","","  16) [STEP2] ユーザログイン後のテスト 自分のユーザ詳細画面のテスト サイドバーの確認 titleフォームが表示される","      \u001b[31mFailure/Error: expect(page).to have_field 'book[title]'\u001b[0m","      \u001b[31m  expected to find field \"book[title]\" that is not disabled but there were no matches\u001b[0m","      \u001b[36m# ./spec/system/02_after_login_spec.rb:346:in `block (4 levels) in <main>'\u001b[0m","","  17) [STEP2] ユーザログイン後のテスト 自分のユーザ詳細画面のテスト サイドバーの確認 titleフォームに値が入っていない","      \u001b[31mFailure/Error: expect(find_field('book[title]').text).to be_blank\u001b[0m","      \u001b[31m\u001b[0m","      \u001b[31mCapybara::ElementNotFound:\u001b[0m","      \u001b[31m  Unable to find field \"book[title]\" that is not disabled\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:129:in `find_field'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","      \u001b[36m# ./spec/system/02_after_login_spec.rb:349:in `block (4 levels) in <main>'\u001b[0m","","  18) [STEP2] ユーザログイン後のテスト 自分のユーザ詳細画面のテスト サイドバーの確認 bodyフォームが表示される","      \u001b[31mFailure/Error: expect(page).to have_field 'book[body]'\u001b[0m","      \u001b[31m  expected to find field \"book[body]\" that is not disabled but there were no matches\u001b[0m","      \u001b[36m# ./spec/system/02_after_login_spec.rb:352:in `block (4 levels) in <main>'\u001b[0m","","  19) [STEP2] ユーザログイン後のテスト 自分のユーザ詳細画面のテスト サイドバーの確認 bodyフォームに値が入っていない","      \u001b[31mFailure/Error: expect(find_field('book[body]').text).to be_blank\u001b[0m","      \u001b[31m\u001b[0m","      \u001b[31mCapybara::ElementNotFound:\u001b[0m","      \u001b[31m  Unable to find field \"book[body]\" that is not disabled\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:129:in `find_field'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","      \u001b[36m# ./spec/system/02_after_login_spec.rb:355:in `block (4 levels) in <main>'\u001b[0m","","  20) [STEP3] 仕上げのテスト サクセスメッセージのテスト 投稿データの新規投稿成功時: 投稿一覧画面から行います。","      \u001b[31mFailure/Error: fill_in 'book[title]', with: Faker::Lorem.characters(number: 5)\u001b[0m","      \u001b[31m\u001b[0m","      \u001b[31mCapybara::ElementNotFound:\u001b[0m","      \u001b[31m  Unable to find field \"book[title]\" that is not disabled\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/actions.rb:91:in `fill_in'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","      \u001b[36m# ./spec/system/03_finishing_touches_spec.rb:53:in `block (3 levels) in <main>'\u001b[0m","","  21) [STEP3] 仕上げのテスト 処理失敗時のテスト 投稿データの新規投稿失敗: 投稿一覧画面から行い、titleを空にする 投稿が保存されない","      \u001b[31mFailure/Error: fill_in 'book[body]', with: @body\u001b[0m","      \u001b[31m\u001b[0m","      \u001b[31mCapybara::ElementNotFound:\u001b[0m","      \u001b[31m  Unable to find field \"book[body]\" that is not disabled\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/actions.rb:91:in `fill_in'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","      \u001b[36m# ./spec/system/03_finishing_touches_spec.rb:128:in `block (4 levels) in <main>'\u001b[0m","","  22) [STEP3] 仕上げのテスト 処理失敗時のテスト 投稿データの新規投稿失敗: 投稿一覧画面から行い、titleを空にする 投稿一覧画面を表示している","      \u001b[31mFailure/Error: fill_in 'book[body]', with: @body\u001b[0m","      \u001b[31m\u001b[0m","      \u001b[31mCapybara::ElementNotFound:\u001b[0m","      \u001b[31m  Unable to find field \"book[body]\" that is not disabled\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/actions.rb:91:in `fill_in'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","      \u001b[36m# ./spec/system/03_finishing_touches_spec.rb:128:in `block (4 levels) in <main>'\u001b[0m","","  23) [STEP3] 仕上げのテスト 処理失敗時のテスト 投稿データの新規投稿失敗: 投稿一覧画面から行い、titleを空にする 新規投稿フォームの内容が正しい","      \u001b[31mFailure/Error: fill_in 'book[body]', with: @body\u001b[0m","      \u001b[31m\u001b[0m","      \u001b[31mCapybara::ElementNotFound:\u001b[0m","      \u001b[31m  Unable to find field \"book[body]\" that is not disabled\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/actions.rb:91:in `fill_in'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","      \u001b[36m# ./spec/system/03_finishing_touches_spec.rb:128:in `block (4 levels) in <main>'\u001b[0m","","  24) [STEP3] 仕上げのテスト 処理失敗時のテスト 投稿データの新規投稿失敗: 投稿一覧画面から行い、titleを空にする バリデーションエラーが表示される","      \u001b[31mFailure/Error: fill_in 'book[body]', with: @body\u001b[0m","      \u001b[31m\u001b[0m","      \u001b[31mCapybara::ElementNotFound:\u001b[0m","      \u001b[31m  Unable to find field \"book[body]\" that is not disabled\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:303:in `block in synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/base.rb:83:in `synchronize'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:292:in `synced_resolve'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/finders.rb:53:in `find'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/node/actions.rb:91:in `fill_in'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/session.rb:778:in `block (2 levels) in <class:Session>'\u001b[0m","      \u001b[36m# /home/ec2-user/.rvm/gems/ruby-2.6.3/gems/capybara-3.36.0/lib/capybara/dsl.rb:58:in `block (2 levels) in <module:DSL>'\u001b[0m","      \u001b[36m# ./spec/system/03_finishing_touches_spec.rb:128:in `block (4 levels) in <main>'\u001b[0m","","  25) [STEP3] 仕上げのテスト アイコンのテスト ヘッダー: ログインしている場合 Usersリンクのアイコンが表示される","      \u001b[31mFailure/Error: is_expected.to have_selector '.fas.fa-users'\u001b[0m","      \u001b[31m  expected to find css \".fas.fa-users\" but there were no matches\u001b[0m","      \u001b[36m# ./spec/system/03_finishing_touches_spec.rb:409:in `block (4 levels) in <main>'\u001b[0m","","  26) [STEP3] 仕上げのテスト アイコンのテスト ヘッダー: ログインしている場合 Booksリンクのアイコンが表示される","      \u001b[31mFailure/Error: is_expected.to have_selector '.fas.fa-book-open'\u001b[0m","      \u001b[31m  expected to find css \".fas.fa-book-open\" but there were no matches\u001b[0m","      \u001b[36m# ./spec/system/03_finishing_touches_spec.rb:412:in `block (4 levels) in <main>'\u001b[0m","","Finished in 17.07 seconds (files took 1.21 seconds to load)","\u001b[31m199 examples, 26 failures\u001b[0m","","Failed examples:","","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:75\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト 投稿一覧画面のテスト サイドバーの確認 titleフォームが表示される\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:78\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト 投稿一覧画面のテスト サイドバーの確認 titleフォームに値が入っていない\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:81\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト 投稿一覧画面のテスト サイドバーの確認 bodyフォームが表示される\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:84\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト 投稿一覧画面のテスト サイドバーの確認 bodyフォームに値が入っていない\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:98\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト 投稿一覧画面のテスト 投稿成功のテスト 自分の新しい投稿が正しく保存される\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:101\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト 投稿一覧画面のテスト 投稿成功のテスト リダイレクト先が、保存できた投稿の詳細画面になっている\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:148\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト 自分の投稿詳細画面のテスト サイドバーの確認 titleフォームが表示される\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:151\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト 自分の投稿詳細画面のテスト サイドバーの確認 titleフォームに値が入っていない\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:154\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト 自分の投稿詳細画面のテスト サイドバーの確認 bodyフォームが表示される\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:157\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト 自分の投稿詳細画面のテスト サイドバーの確認 bodyフォームに値が入っていない\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:171\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト 自分の投稿詳細画面のテスト 投稿成功のテスト 自分の新しい投稿が正しく保存される\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:291\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト ユーザ一覧画面のテスト サイドバーの確認 titleフォームが表示される\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:294\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト ユーザ一覧画面のテスト サイドバーの確認 titleフォームに値が入っていない\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:297\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト ユーザ一覧画面のテスト サイドバーの確認 bodyフォームが表示される\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:300\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト ユーザ一覧画面のテスト サイドバーの確認 bodyフォームに値が入っていない\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:345\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト 自分のユーザ詳細画面のテスト サイドバーの確認 titleフォームが表示される\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:348\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト 自分のユーザ詳細画面のテスト サイドバーの確認 titleフォームに値が入っていない\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:351\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト 自分のユーザ詳細画面のテスト サイドバーの確認 bodyフォームが表示される\u001b[0m","\u001b[31mrspec ./spec/system/02_after_login_spec.rb:354\u001b[0m \u001b[36m# [STEP2] ユーザログイン後のテスト 自分のユーザ詳細画面のテスト サイドバーの確認 bodyフォームに値が入っていない\u001b[0m","\u001b[31mrspec ./spec/system/03_finishing_touches_spec.rb:47\u001b[0m \u001b[36m# [STEP3] 仕上げのテスト サクセスメッセージのテスト 投稿データの新規投稿成功時: 投稿一覧画面から行います。\u001b[0m","\u001b[31mrspec ./spec/system/03_finishing_touches_spec.rb:131\u001b[0m \u001b[36m# [STEP3] 仕上げのテスト 処理失敗時のテスト 投稿データの新規投稿失敗: 投稿一覧画面から行い、titleを空にする 投稿が保存されない\u001b[0m","\u001b[31mrspec ./spec/system/03_finishing_touches_spec.rb:134\u001b[0m \u001b[36m# [STEP3] 仕上げのテスト 処理失敗時のテスト 投稿データの新規投稿失敗: 投稿一覧画面から行い、titleを空にする 投稿一覧画面を表示している\u001b[0m","\u001b[31mrspec ./spec/system/03_finishing_touches_spec.rb:140\u001b[0m \u001b[36m# [STEP3] 仕上げのテスト 処理失敗時のテスト 投稿データの新規投稿失敗: 投稿一覧画面から行い、titleを空にする 新規投稿フォームの内容が正しい\u001b[0m","\u001b[31mrspec ./spec/system/03_finishing_touches_spec.rb:144\u001b[0m \u001b[36m# [STEP3] 仕上げのテスト 処理失敗時のテスト 投稿データの新規投稿失敗: 投稿一覧画面から行い、titleを空にする バリデーションエラーが表示される\u001b[0m","\u001b[31mrspec ./spec/system/03_finishing_touches_spec.rb:408\u001b[0m \u001b[36m# [STEP3] 仕上げのテスト アイコンのテスト ヘッダー: ログインしている場合 Usersリンクのアイコンが表示される\u001b[0m","\u001b[31mrspec ./spec/system/03_finishing_touches_spec.rb:411\u001b[0m \u001b[36m# [STEP3] 仕上げのテスト アイコンのテスト ヘッダー: ログインしている場合 Booksリンクのアイコンが表示される\u001b[0m","",""],"id":2}]]},"ace":{"folds":[],"scrolltop":6504,"scrollleft":0,"selection":{"start":{"row":314,"column":10},"end":{"row":314,"column":35},"isBackwards":true},"options":{"guessTabSize":true,"useWrapMode":false,"wrapToView":true},"firstLineState":{"row":405,"state":"start","mode":"ace/mode/text"}},"timestamp":1655880951556,"hash":"38d3a13399a7960b7a231cc328b1a118a9d79e04"}